Package.swift

```diff
@@ -7,22 +7,30 @@ let package = Package(
     name: "EmbeddedProvision",
     platforms: [
         .iOS(.v12),
-        .macOS(.v10_15)
+        .macOS(.v10_15),
     ],
     products: [
         .library(
             name: "EmbeddedProvision",
             targets: ["EmbeddedProvision"]
-        ),
+        )
```

Comment:
Thanks for cleaning up that comma, but why did you move it to line 10? ;)

Reply:
I didn't do anything - my editor auto formatted :)
Do you have some formatting settings to apply?

Reply:
Indeed, running `swift format -i --recursive Sources Tests Package.swift` applies all that. I'll automate formatting in a pre-commit hook in a seperate PR, and will reformat everything (lot's of changes in test files).

It's the same formatting that's built in in Xcode 16 via `ctrl+shift+i`.